dbt_platform_helper/commands/config.py

```diff
@@ -76,7 +76,10 @@ def deployment():
     tool_versions()
     click.secho("Checking addons templates versions...", fg="blue")
 
-    app_version, app_released_version = versioning.get_app_versions()
+    versions = versioning.get_platform_helper_versions()
+    app_version = versions["package_version"]
+    app_released_version = versions["platform_helper_file_version"]
```

Comment:
I think these might be more intuitive names...
```suggestion
    local_version = versions["package_version"]
    platform_helper_file_version = versions["platform_helper_file_version"]
```